server/neighborhood.js

```diff
@@ -0,0 +1,136 @@
+class neighborhood{
+    constructor(info){
+        this.median_value=info[0];
+        this.median_income=info[1];
+        this.median_age=info[2];
+        this.total_rooms=info[3];
+        this.total_bedrooms=info[4];
+        this.population=info[5];
+        this.households=info[6];
+        this.latitude=info[7];
+        this.longitude=info[8];
+        this.distance_to_coast=info[9];
+        this.distance_to_LA=info[10];
+        this.distance_to_SD=info[11];
+        this.distance_to_SJ=info[12];
+        this.distance_to_SF=info[13];
+        if(info[10]>info[11]&&info[10]>info[12]&&info[10]>info[13]){
+            this._closest_metro="Los Angeles";
+        }
+        else if(info[11]>info[10]&&info[11]>info[12]&&info[11]>info[13]){
+            this._closest_metro="San Diego";
+        }
+        else if(info[12]>info[11]&&info[12]>info[10]&&info[12]>info[13]){
+            this._closest_metro="San Jose";
+        }
+        else if(info[13]>info[11]&&info[13]>info[12]&&info[13]>info[10]){
```

Comment:
Interesting. Didn't catch this the first time around. Should we cache this information or have it as a getter? Doesn't matter really to me.

Reply:
I was just thinking that way there was less computation for each function call of that